FSL Melodic - Form UI improvements

remibernard opened the pull request:
Issues #7 and #14

remibernard committed:
Issue #7 - Dynamic fields in task form

All requested components are implemented via a small JS snippet at the
end of the file. Note that most option groups are now in separate UL
elements.

This changeset also removes leftover EOL whitespace.

remibernard committed:
Issue #14 - Add the possiblity to disable some CSV entries

Implemented using a simple checkbox which switches the hidden
inputs to disabled (prevents sending them with the form).

glatard commented:
Could you "merge" the registration checkbox with the registration titles (as discussed)?

remibernard committed:
Moved registration section checkboxes

The checkboxes are now next to the registration section names. Makes for
a sightly simpler interface.

glatard commented:
The warning on "registration to main structural" is broken. When the corresponding parameter is not enabled, the UI must:
- enable the parameter
- show the warning.

remibernard committed:
Forced Registration to main structural option (:reghighres_yn)

Now always active (on form load) and displays a warning if it wasn't
already. Also reformatted one of the hard-to-read option value list.
(Shouldn't those be made into radio button groups?)

natacha-beck commented:
Tristan already take a look I take a second one before merging, should be good.
